app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -30,7 +30,7 @@
     = link_to t('jobs.submit_listing.button'), school_job_publish_path(@vacancy.id), method: :post, class: 'govuk-button govuk-button--secondary submit-listing-without-preview-gtm', id: 'vacancy-review-submit'
 
     %br
-    = render 'hiring_staff/vacancies/back_to_manage_jobs'
+    = link_to t('buttons.back_to_manage_jobs'), jobs_with_type_school_path('draft', flash_success: t('messages.jobs.draft_saved_html', job_title: @vacancy.job_title)), class: 'govuk-link govuk-link--no-visited-state'
```

Comment:
Passing the content of the message could introduce a security problem here. I think you can pass just a parameter that flags that specific message to that specific view. We do not need to make it general